src/MultiplicationTable.ts

```diff
@@ -1,5 +1,30 @@
-export class MultiplicationTable {
-  public render(start: number, end: number): string{
-    return '1*1=1'
+export function render(start: number, end: number): string {
+  const isNotStartGreaterThanEnd = !isStartGreaterThanEnd(start,end)
+  const isValidate = isInRange(start, end)
+  if (isNotStartGreaterThanEnd&&isValidate) {
+    return createTable(start,end)
+  }else{
+    return ''
   }
+
+}
+
+export function isStartGreaterThanEnd(start: number, end: number) {
+  return start > end
+}
+export function isInRange(start: number, end: number): boolean {
+  const range = Array.from(Array(10).keys()).map(x => x+1)
+  return range.findIndex(element => element === start) >= 0 && range.findIndex(element => element === end) >= 0
+}
```

Comment:
can use Math.floor() to check integer

Reply:
that's a good idea

Reply:
Considered decimals, improves robustness